src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -7,45 +7,100 @@ import {
   LARGE_SCREEN_MEDIA_QUERY,
   MEDIUM_SCREEN_MEDIA_QUERY,
 } from 'utils/mediaQueries'
+import { Button } from 'components/buttons'
 import { quoteDetailsHasAddress } from '../utils'
 import { getAddress } from '../Checkout/InsuranceSummaryDetails'
+import { DetailsModal } from './DetailsModal'
 
 type Props = {
   offerData: OfferData
+  refetchOfferData: () => Promise<void>
 }
 
 const Wrapper = styled.div`
-  padding: 2.5rem 0 1rem;
-  color: ${colorsV3.white};
+  padding: 3.5rem 0 0;
+  color: ${colorsV3.gray900};
 
   ${LARGE_SCREEN_MEDIA_QUERY} {
     padding-top: 0;
+    color: ${colorsV3.gray100};
   }
 `
 const Headline = styled.h1`
-  margin: 0;
+  margin-bottom: 0.5rem;
   text-transform: uppercase;
-  font-size: 1rem;
+  font-size: 0.875rem;
+  line-height: 1.375rem;
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    font-size: 1rem;
+    line-height: 1.5rem;
+  }
 `
 const OfferInfoWrapper = styled.div`
-  padding-bottom: 2rem;
+  padding-bottom: 1.5rem;
 `
 const NameAndCoInsured = styled.div`
-  font-size: 2rem;
+  font-size: 1.5rem;
+  line-height: 2rem;
+  display: inline-block;
+  margin-right: 0.5rem;
 
   ${MEDIUM_SCREEN_MEDIA_QUERY} {
     font-size: 3rem;
+    line-height: 3.5rem;
+  }
+
+  &:after {
+    content: ',';
+    display: inline;
+
+    ${MEDIUM_SCREEN_MEDIA_QUERY} {
+      display: none;
+    }
   }
 `
+
 const Address = styled.div`
-  font-size: 1.375rem;
+  font-size: 1.5rem;
+  line-height: 2rem;
+  display: inline-block;
 
   ${MEDIUM_SCREEN_MEDIA_QUERY} {
     font-size: 2.5rem;
+    display: block;
+  }
+`
+
+const EditDetailsButton = styled(Button)`
+  border: 1px solid ${colorsV3.gray900};
+  color: ${colorsV3.gray900};
+  width: 100%;
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    border: 1px solid ${colorsV3.gray100};
+    color: ${colorsV3.gray100};
+    width: auto;
+  }
+
+  :hover {
```

Comment:
Should we be consistent here and write:

```suggestion
  &:hover {
```